app/views/admin/users/_form.html.erb

```diff
@@ -48,7 +48,13 @@
                     <% end %>
                 </div>
             </div>
-
+            <h3 class="p3"><%= t('users.daily_summary') %></h3>
```

Comment:
Using short form `t('.')` enables deeper namespacing (which is a good thing).

Reply:
Scope is `users.form` and `daily_summary` is already defined in `users` scope (same translation is used on profile page). Not going to change this one because in my opinion it's not good idea to duplicate translations, is it?

Reply:
It might make more sense to use `admin` namespace, since this is what it represents.

Reply:
> In case if the key is too generic hard to track the relationship between the source and actual place of display on the page

This is what I meant. Taken from first link in Google: https://dev.to/omaiboroda/three-ways-to-name-i18n-translation-keys-2fed

Reply:
Rails translations seem have nothing to do with Gettext, where such practice is used (one single translation "key").

Reply:
Alright, makes sense. Making changes.